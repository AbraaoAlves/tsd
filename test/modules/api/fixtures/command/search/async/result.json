{
  "nameMatches": [
    {
      "path": "async/async.d.ts",
      "project": "async",
      "name": "async",
      "def": {
        "path": "async/async.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "async/async.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "async/async.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "async/async.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    }
  ],
  "written": {}
}