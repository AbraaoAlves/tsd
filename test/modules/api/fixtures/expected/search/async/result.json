{
  "selection": [
    {
      "path": "async/async.d.ts",
      "key": "async/async.d.ts-1efa27f49add3fcb68450f2c1cfa8fc581ec2506",
      "solved": false,
      "commit": {
        "commitSha": "1efa27f49add3fcb68450f2c1cfa8fc581ec2506"
      },
      "blob": {
        "sha": "12aa4123f3e68a5ef9a45108c55abd9e1b645851"
      },
      "dependencies": []
    }
  ],
  "nameMatches": [
    {
      "path": "async/async.d.ts",
      "project": "async",
      "name": "async",
      "head": {
        "path": "async/async.d.ts",
        "key": "async/async.d.ts-1efa27f49add3fcb68450f2c1cfa8fc581ec2506",
        "solved": false
      },
      "history": []
    }
  ],
  "written": {}
}